### PR TITLE
feat: Add auto-approve bot config

### PR DESCRIPTION
**Pull request by bourgeoisor**
This PR adds config for the auto-approve bot (documented here: https://github.com/googleapis/repo-automation-bots/tree/main/packages/auto-approve) which will approve Renovate PRs that follows very strict conventions and rules (e.g. only patches and minor bumps, all tests must be passing, etc.) Full details of these rules can be found in the previously linked README.

### CODE REVIEW

**Review by NimJay — APPROVED** [review]

File: .github/auto-approve.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://github.com/googleapis/repo-automation-bots/tree/main/packages/auto-approve
+processes:
+  - "JavaDependency"
```
Comment (NimJay) [inline .github/auto-approve.yml:17]: What about `NodeDependency`?

Comment (bourgeoisor) [reply .github/auto-approve.yml]: Whoops, good catch!

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [****](http://)
- **Embedded H2** DB backed deployment: [****](http://)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**35.232.184.87**](http://35.232.184.87)
- **Embedded H2** DB backed deployment: [**34.136.54.241**](http://34.136.54.241)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**35.232.184.87**](http://35.232.184.87)
- **Embedded H2** DB backed deployment: [**34.136.54.241**](http://34.136.54.241)

**Pull request merged**